.eslintrc.js

```diff
@@ -32,4 +32,58 @@ module.exports = {
     '@next/next/no-img-element': ['off'],
     'linebreak-style': ['error', os.platform() === 'win32' ? 'windows' : 'unix'],
   },
+  overrides: [
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      env: {
+        es2021: true,
+        node: true,
+        jest: true,
+      },
+      extends: [
+        'next',
+        'airbnb',
+        'airbnb-typescript',
+        'plugin:cypress/recommended',
+        'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+      ],
+      parser: '@typescript-eslint/parser',
+      parserOptions: {
+        ecmaVersion: 12,
+        sourceType: 'module',
+        project: ['./tsconfig.json'],
+      },
```

Comment:
why is this section here in the overrides section? i think you can just updated the existing fields above

Reply:
but the `overrides` section shouldn't be used - rather all these unique additions that you made should be merged in the original section of this file